backend/.env.example

```
@@ -9,3 +10,6 @@ PORT=8080
 #   https://wip-frontend.uozumi05.workers.dev  (workers.dev)
 #   https://wip-frontend.uomi.dev              (カスタムドメイン)
 CORS_ALLOW_ORIGINS="http://localhost:3000,http://localhost:5173"
+
+# ゲストCookieの署名鍵（本番では `openssl rand -base64 32` などで生成したランダム値に置き換える）
+GUEST_SESSION_SECRET="replace-with-a-long-random-secret"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)